docs/examples/viz_network_animated_billboard.py

```diff
@@ -0,0 +1,349 @@
+                              fs_dec=fs_dec,
+                              fs_impl=fake_sphere
+                              )
+
+###############################################################################
+# Preparing editable geometry for the nodes
+
+vtk_centers_geometry = vtk_array_from_actor(nodes_actor, array_name="center")
+centers_geometry = VN.vtk_to_numpy(vtk_centers_geometry)
+centers_geometryOrig = np.array(centers_geometry)
+centers_length = centers_geometry.shape[0] / positions.shape[0]
+
+
+vtk_verts_geometry = vtk_vertices_from_actor(nodes_actor)
+verts_geometry = VN.vtk_to_numpy(vtk_verts_geometry)
+verts_geometryOrig = np.array(verts_geometry)
+verts_length = verts_geometry.shape[0] / positions.shape[0]
+
+
+###############################################################################
+# Defining timer callback and layout iterator. Helios-core
+# is used to dynamically layout the network
+
+velocities = np.zeros((vertices_count, 3), dtype=np.float32)
+edgesArray = np.ascontiguousarray(np.array(edges), dtype=np.uint64)
+layout = None
+
+
+def new_layout_timer(showm, edges_list, vertices_count,
+                     max_iterations=1000, vertex_initial_positions=None):
+    global layout
+    counter = 0
+    if(vertex_initial_positions is not None):
+        positions[:] = np.array(vertex_initial_positions)
+    else:
+        positions[:] = view_size * \
+            np.random.random((vertices_count, 3)) - view_size / 2.0
+
+    viscosity = 0.30
+    a = 0.0005
+    b = 1.0
+
+    layout = helios.FRLayout(edgesArray, positions,
+                             velocities, a, b, viscosity)
+    layout.start()
+    framesPerSecond = []
+
+    def _timer(_obj, _event):
+        nonlocal counter, framesPerSecond
+        counter += 1
+        framesPerSecond.append(scene.frame_rate)
+
+        centers_geometry[:] = np.repeat(positions, centers_length, axis=0)
+        verts_geometry[:] = verts_geometryOrig + centers_geometry
+
+        edges_positions = VN.vtk_to_numpy(
+            lines_actor.GetMapper().GetInput().GetPoints().GetData())
+        edges_positions[::2] = positions[edges_list[:, 0]]
+        edges_positions[1::2] = positions[edges_list[:, 1]]
+
+        lines_actor.GetMapper().GetInput().GetPoints().GetData().Modified()
+        lines_actor.GetMapper().GetInput().ComputeBounds()
+        vtk_verts_geometry.Modified()
+        vtk_centers_geometry.Modified()
+        update_actor(nodes_actor)
+
+        if(selectedNode is not None):
+            selectedActor.SetPosition(positions[selectedNode])
+
+        nodes_actor.GetMapper().GetInput().GetPoints().GetData().Modified()
+        nodes_actor.GetMapper().GetInput().ComputeBounds()
+        selectedActor.GetMapper().GetInput().ComputeBounds()
+        selectedActor.GetMapper().GetInput().GetPoints().GetData().Modified()
+        showm.scene.ResetCameraClippingRange()
+        showm.render()
+
+        if counter >= max_iterations:
+            showm.exit()
```

Comment:
why you do not stop the layout here?

Reply:
The user may close the window before the counter reaches the maximum number of iterations, that is why the stopLayout is not in the timer function. Any suggestions?